app/routes/podcast.tsx

```diff
@@ -1,11 +1,47 @@
 //podcasts.apple.com/ph/podcast/trash-turtle-football/id1586577488
+import type { LoaderArgs } from "@remix-run/node";
+
+import { getEpisodes } from "~/models/episode.server";
+import type { Episode } from "~/models/episode.server";
+
+import { superjson, useSuperLoaderData } from "~/utils/data";
+
+type LoaderData = {
+  episodes: Episode[];
+};
+
+export const loader = async ({ request }: LoaderArgs) => {
+  const episodes = await getEpisodes();
+
+  return superjson<LoaderData>(
+    { episodes },
+    { headers: { "x-superjson": "true" } }
+  );
+};
+
 
 export default function Records() {
+  const { episodes } = useSuperLoaderData<typeof loader>();
   return (
-    <div>
-      <a href="//podcasts.apple.com/ph/podcast/trash-turtle-football/id1586577488">
-        Go listen and subscribe for now.
-      </a>
+    <div className="text-center md:text-left">
+
+      <h2 id="Title">Trash Turtle Football</h2>
+
+      <img alt="podcast image" src="/ttflogowords.jpg" className="h-36 w-36 lg:w-56 lg:h-56 mx-auto md:ml-4 mb-4 float-none md:float-right"></img>
```

Comment:
```suggestion
      <img alt="Trash Turtle Football podcast logo" src="/ttflogowords.jpg" className="h-36 w-36 lg:w-56 lg:h-56 mx-auto md:ml-4 mb-4 float-none md:float-right"></img>
```

Nit: Just potentially a better alt text option to be more clear for screen readers.